### ISSUE TITLE
MSID: 90735 Version: 1 DOI: osf.io/r49sy

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1764]
"msas": "Neuroscience"
"msid": "90735"
"preprintDoi": "10.31234/osf.io/r49sy"
"Reviewed Preprint posted": "2023-11-21"
"Sent for peer review": "2023-08-11"
"Posted to PsyArXiv": "2023-07-05" (link: "Go to PsyArXiv": "https://www.osf.io/preprints/psyarxiv/r49sy/")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=90735)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/90735/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.31234/osf.io/r49sy)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [x] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/90735/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/90735)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/90735
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/90735
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/90735/v1/90735-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/90735)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by JGilbert-eLife** [comment]
This equation has been tagged as a figure:

![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/1785305/a1f04e1b-01c7-448b-b38b-893644e6c006)

**Comment by fred-atherden** [comment]
It's unfortunate but I don't think the workaround is going to be possible for this one so we'll have to treat this as a platform issue (it is fundamentally a problem with Encoda) and proceed

**Comment by fred-atherden** [comment]
Reviews are not being returned by EPP or present in DocMap - needs further investigation.

**Comment by fred-atherden** [comment]
@hazalCiplak - do you know why the reviews haven't been pulled into the docmap here? 

I see them at https://via.hypothes.is/https://osf.io/preprints/psyarxiv/r49sy/

And the details are included in this spreadsheet - https://docs.google.com/spreadsheets/d/15QcK8w-ssB7109RQEDtFpJPZ0J5HTGxoHa_2TtpMBbg/edit#gid=1336081641

**Comment by HazalCiplak** [comment]
It seems `uri` in hypothesis data is in a different format for this one in hypothesis data 🤔. I will update the spreadsheet accordingly.

<img width="1011" alt="Screenshot 2023-11-24 at 09 50 24" src="https://github.com/elifesciences/enhanced-preprints-import/assets/6259479/06f2106b-c6c9-4774-98dd-db019f461006">

**Comment by fred-atherden** [comment]
Thanks Hazal. Huh  - that's odd. I wonder why it's only affected this one and not other OSF preprints? Maybe OSF recently changed their URL format?

@acollings FYI - it looks like we need to use the format https://osf.io/preprints/psyarxiv/r49sy instead of https://psyarxiv.com/r49sy for psyArXiv/OSF when adding URLs to [this spreadsheet.](https://docs.google.com/spreadsheets/d/15QcK8w-ssB7109RQEDtFpJPZ0J5HTGxoHa_2TtpMBbg/edit#gid=1336081641)

**Comment by HazalCiplak** [comment]
Hi @acollings, @fred-atherden,

Apologies for the confusion, but we shouldn't change the other entries. The field in the GSheet should have the same format as what we have in Hypothesis data. This is because we are joining the data based on the URL, and if they are different, it will prevent us from making the connection between Hypothesis and GSheet. The example with the format (https://osf.io/preprints/psyarxiv/r49sy) was the only one with that format. The others should remain as they are. I have created a separate ticket to investigate how to improve the join key, maybe based on just ID: https://github.com/elifesciences/data-hub-issues/issues/814. I will also rollback the changes in the GSheet. Thank you!

**Comment by HazalCiplak** [comment]
Somehow related question, we have the row below in the document, and we don't have osf_preprint_url. When I check the Hypothesis data by OSF ID to add the URL, there is no result coming. So, can there be a mistake with the preprint_doi here?

manuscript_id | long_manuscript_identifier | preprint_doi | preprint_doi_version | preprint_published_at_date | preprint_server | osf_preprint_id | osf_preprint_url |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
92683 | eLife-RP-RA-2023-92683 | 10.31219/osf.io/s6thq | 1 | 2023-10-11 | OSF | s6thq |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |

**Comment by acollings** [comment]
Thanks @HazalCiplak For 92683, https://doi.org/10.31219/osf.io/s6thq / https://osf.io/s6thq/ is correct, but the paper is under review so there aren't hypothes.is reviews yet.

**Comment by HazalCiplak** [comment]
Typically, yes, we can expect them to be there. I've found memory issues on the cluster preventing them from running, so I've manually triggered them. The DocMaps should be available soon. Thank you for your patience.

**Comment by fred-atherden** [comment]
Nice one - thanks Hazal!

**Comment by fred-atherden** [comment]
@HazalCiplak, sorry, I'm still not seeing the reviews on EPP/in the docmap. Please could you take a look again when you have a chance? Thanks

**Comment by HazalCiplak** [comment]
Ah sorry to hear that and thanks for letting me know! I will look into it now.

**Comment by HazalCiplak** [comment]
There was a filter in the code screening OSF preprints based on URL format (not includes the new version), causing us to miss evaluations for this one. I've now adjusted the filter, and the DocMap will be updated shortly. My apologies for not catching this earlier.

**Comment by HazalCiplak** [comment]
The DocMap has been updated and evaluations are available.
